applications/jupyter/workloads.tfvars

```diff
@@ -13,14 +13,15 @@
 # limitations under the License.
 
 ##common variables
-## Need to pull this variables from tf output from previous platform stage
+## Need to pull this variables from tf output from previous infrastructure stage
 project_id = "<your project ID>"
 
 ## this is required for terraform to connect to GKE master and deploy workloads
-create_cluster        = false # this flag will create a new standard public gke cluster in default network
+create_cluster        = false # this flag will create a new autopilot public gke cluster in the default network
 cluster_name          = "ml-cluster"
 cluster_location      = "us-central1"
 cluster_membership_id = "" # required for private cluster, defaults to `cluster_name`
+autopilot_cluster     = true
```

Comment:
all of the flags in this section are related to the create cluster (cluster name, location, type, fleet membership). I've moved the flag up one line, but since I expect that it's more likely to change the cluster name or location than the type, I'd prefer to leave it below those two variables.

Reply:
Restructured a bit more so that the terraform lint check would pass.